changelog_unreleased/typescript/13860.md

```diff
@@ -0,0 +1,18 @@
+#### Fix union type should be printed in the multi-line variant when there are comments (#13860 by @PerfectPan)
+
+<!-- prettier-ignore -->
+```tsx
+// Input
+type FooBar = 
+  | Number // this documents the first option
+  | void // this documents the second option
+  ;
```

Comment:
Can you add this case to test?

Reply:
OK，I will do it when I get off work today

Reply:
Done. [`dcc74d9` (#13860)](https://github.com/prettier/prettier/pull/13860/commits/dcc74d9459cb80e4a016374918f0daed398dba31)